ptvsd/wrapper.py

```diff
@@ -1350,6 +1357,52 @@ def on_exceptionInfo(self, request, args):
                      'source': exc.source},
         )
 
+    # Custom ptvsd message
+    def on_ptvsd_systemInfo(self, request, args):
```

Comment:
Should we expose this feature via capabilities `supportsPtvsdSystemInfo`? Also, are we making use of this in VS?

Reply:
Not yet, but I was hoping that Attach to Process could use some of these :)

The problem with capabilities is that, so far as I can see, the schema doesn't say that we can add our things there? And in capabilities, it would be observable by everyone, not just our bits on the IDE sides (whereas with the new message type, you only see it if you issue the request in the first place). So I'm not sure if it's safe to put it there.